README.md

```diff
@@ -219,3 +220,78 @@ Cols: []
 -----------------------------------------
 Excel files xxx.xlsx and yyy.xlsx match
 </pre>
+
+## Unified Diff output format
+* Diffs are reported in the "unified diff" style, with no surrounding context (_i.e._, a la `diff -U0`).
+* Each sheet containing a diff or an extra cell begins with a header as follows:
+<pre>
+		--<FileName1>!<SheetName>
+		++<FileName2>!<SheetName>
+</pre>
+* Each row containing a diff or an extra cell begins with a line that identifies the specific cell range as follows:
```

Comment:
README could still be more clear, with the phrase like - contiguous series of cells in one row

Reply:
Will do.

Reply:
Done.